sklearn/utils/tests/test_plotting.py

```diff
@@ -61,3 +65,60 @@ def test_validate_score_name(score_name, scoring, negate_score, expected_score_n
 )
 def test_inverval_max_min_ratio(data, lower_bound, upper_bound):
     assert lower_bound < _interval_max_min_ratio(data) < upper_bound
+
+
+@pytest.mark.parametrize(
+    "default_kwargs, user_kwargs, expected",
+    [
+        (
+            {"color": "blue", "linewidth": 2},
+            {"linestyle": "dashed"},
+            {"color": "blue", "linewidth": 2, "linestyle": "dashed"},
+        ),
+        (
+            {"color": "blue", "linestyle": "solid"},
+            {"c": "red", "ls": "dashed"},
+            {"color": "red", "linestyle": "dashed"},
+        ),
+        (
+            {
+                "label": "Chance level (AUC = 0.5)",
+                "color": "k",
+                "linestyle": "--",
+            },
```

Comment:
```suggestion
            {"label": "xxx", "color": "k","linestyle": "--"},
```

Reply:
Let's make the dictionary fit on a single line such that we can quickly parse the parametrization for each use-case. We don't really care at the end about the value of "label"